superset-frontend/src/views/App.tsx

```diff
@@ -83,13 +83,9 @@ const App = () => (
         {routes.map(({ path, Component, props = {}, Fallback = Loading }) => (
           <Route path={path} key={path}>
             <Suspense fallback={<Fallback />}>
-              <Layout.Content>
-                <div style={{ padding: '16px' }}>
-                  <ErrorBoundary>
-                    <Component user={bootstrapData.user} {...props} />
-                  </ErrorBoundary>
-                </div>
-              </Layout.Content>
+              <ErrorBoundary style={{ margin: '16px' }}>
```

Comment:
wanted to do `theme.sizeUnit * 4` but theme isn't in context here so bailed and went for hard-coded